### ISSUE TITLE
My wallet is synced and up to date it says node connected and then nothing happens. I can't open the wallet. I am using the latest version for Mac OSX

### ISSUE THREAD

**Issue by Dbabcock001** [ethereum/mist#3868]
* [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0`
Number of blocks synchronized: 0
```

**Comment by ncsolar** [comment]
Same issue here, Mist stays on splash screen "Ethereum node connected"

**Comment by ryanio** [comment]
Hey, we just released a new version (`0.11.0`) that has new syncing features - please give it a try! If you have any new problems that you can reproduce, please create a new issue with the details on how 
we can help. Thanks!
